### ISSUE TITLE
Search result - change message and add an option to search the archive/unarchived bookmarks

### ISSUE THREAD

**Issue by Export33** [sissbruecker/linkding#84]
#### Foreword

I´m on a run and using linkding more and more. Please let me know if my requests are too much/nutpicking.

# Actual behaviour

The message says `You have no boomarks yet` if no search result is found which is technically not true.

![grafik](https://user-images.githubusercontent.com/25232960/108563786-577ef900-7302-11eb-82b5-81a042a65109.png)

# Expected behaviour

You may want to specify the message:
`You have no bookmarks yet.` if there is really no bookmark.
`Your search - SEARCHTERM - did not match any bookmarks.` if there are bookmarks but none fits the searchterm
`Your search - SEARCHTERM - did not match any archived bookmarks.`  if there are archived bookmarks but none fits the searchterm

Additionally you could add an option to search the `Archive`/`Bookmarks` instead.

# Example

![grafik](https://user-images.githubusercontent.com/25232960/108564219-0f140b00-7303-11eb-9c98-d391949a703d.png)

**Comment by sissbruecker** [comment]
Makes sense, but low prio. I assume that every user knows whether they have bookmarks or not, so while the message is confusing, one can probably infer what the message means.

**Comment by gabrielwhite** [comment]
I just came to this message after triggering a search in Linkding from the Chrome extension, and thought that I'd lost all my data. So there are use cases that might be concerning for regular users.

Maybe this can be fixed with a minor enhancement and change of wording.

"No bookmarks found (KEYWORD)" - the "found (KEYWORD)" is only shown when in the context of search.

"Bookmarks can be added <ins>directly</ins>, using the <ins>bookmarklet</ins>, or <ins>imported</ins>" - this can be shown permanently in both cases.
